manage.js

```diff
@@ -88,6 +83,11 @@ function initGlobalEvents() {
     'manage.newUI.targets',
   ]);
 
+  $$('[data-filter]').forEach(el => {
```

Comment:
To attach to the parent element we'd have to use `.closest('fieldset')`, which is dependent on the tag name, or otherwise specify the parent element explicitly. I think it's simpler to maintain just a `data-filter` attribute on the affected element directly.